Practice/fizz buzz

YudaiNoda3576 opened the pull request:
## 目的
プラクティスFizzBuzz問題(ruby)の課題を提出すること

## 達成条件
レビューを頂きレビュワーからマージ許可をもらうこと

## 実装の概要
* 1から20までの数をプリントするプログラム
* 3の倍数のときは数の代わりに｢Fizz｣とプリントする
* 5の倍数のときは｢Buzz｣とプリントする
* 3と5両方の倍数の場合には｢FizzBuzz｣とプリントする

## レビューして欲しいところ
* 使用している関数が適切か否か
* クラスの分割方法が適切か否か
* メソッドの用法は適切か否か
* メソッド・クラス名などの命名に問題はないか

## ここはこうしたけどこの点で問題はないだろうか
* クラスを使用してみたが、使用方法はこちらで間違っていないでしょうか？
* FIZZBUZZの出力条件を15の倍数とすると記述の順番に依存するため、3の倍数かつ5の倍数のような条件の記述としたが適切でしょうか？

YudaiNoda3576 committed:
feat

FIZZBUZZ問題の回答を作成

YudaiNoda3576 committed:
refactor

クラスを作成し、メソッドとして呼び出せるように修正

YudaiNoda3576 committed:
fix

出力する文字列のタイポを修正

YudaiNoda3576 committed:
fix

メソッド名が不適切だったため修正

JunichiIto reviewed (commented):
動画でレビューしたので別途リンクを送ります〜。

YudaiNoda3576 committed:
refactor

initializeの削除、クラスの外でメソッドを呼び出すように修正、出力する文字列を定数から除外、convert_fizzbuzzを実行するメソッドの名前をrunに変更

YudaiNoda3576 committed:
refactor

convert_fizzbuzzの条件分岐を修正

JunichiIto commented on 01.fizzbuzz/fizz_buzz.rb:
```diff
@@ -0,0 +1,23 @@
+    end
+  end
+end
+  FizzBuzz.new.run(1,20)
```
```suggestion
FizzBuzz.new.run(1, 20)
```

クラス構文の外なのでインデントはなくしましょう。
あとカンマの後ろはスペースを入れることが多いです。

JunichiIto reviewed (approved):
1点コメントしましたが大きな問題ではないのでこれでOKです！
